pkg/k8s/apps/translate.go

```diff
@@ -337,6 +337,32 @@ func TranslateEnvVars(c *apiv1.Container, rule *model.TranslationRule) {
 			c.Env = append(c.Env, apiv1.EnvVar{Name: envvar.Name, Value: value})
 		}
 	}
+
+	c.Env = append(c.Env,
```

Comment:
would it make more sense to execute this on the SSH session instead? that way a) we have more information on the available shell sessions and b) we don't have to "expose" this implementation detail in the kubernetes manifest?